portal/src/graphql/portal/UserInterfaceScreen.tsx

```diff
@@ -58,9 +58,9 @@ const UserInterfaceContent: React.FC<UserInterfaceContentProps> = function UserI
   return (
     <div className={styles.root}>
       <NavBreadcrumb items={navBreadcrumbItems} />
-      <Label>
+      <Text className={styles.title}>
```

Comment:
This looks strange, since the field labels on other screens are bolded.

Reply:
Changed back to Label.